projects/services/consensus-service-typescript/src/transaction-pool-service.ts

```diff
@@ -67,6 +67,12 @@ export default class TransactionPoolService extends Service {
     logger.debug(`getAllPendingTransactions() . returns ${JSON.stringify(rpc.res)}`);
   }
 
+  @Service.RPCMethod
+  public async getTransactionStatus(rpc: types.GetTransactionStatusContext) {
+    const txid = rpc.req.txid;
+    this.pendingTransactionPool.getTransactionStatus(txid);
```

Comment:
And if it's no longer pending / was committed a year ago?

Reply:
No support for that (for now)